src/Utils.hs

```diff
@@ -215,15 +218,8 @@ sendFileChan :: ChannelId -> T.Text -> FilePath -> DiscordHandler ()
 sendFileChan c name fp = do
     mFileContent <- liftIO $ safeReadFile fp
     case mFileContent of
-        Nothing          -> do
-<<<<<<< HEAD
-            _ <- liftIO $ putStrLn $ "[WARN] Couldn't load file: " <> fp
```

Comment:
Please can we keep this error message, I added it yesterday cos we keep on having problems with asset loading and it's nice to see what we're trying to read

Reply:
Seems to have been accidentally removed during a merge, fixed now.